aws/rust-runtime/aws-config/integration-tests/eks-credentials/lib/eks-credentials-stack.ts

```diff
@@ -0,0 +1,39 @@
+import * as cdk from '@aws-cdk/core';
+import * as eks from '@aws-cdk/aws-eks';
+import * as dynamodb from '@aws-cdk/aws-dynamodb';
+export class EksCredentialsStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+
+    // create a cluster
+    const cluster = new eks.Cluster(this, 'hello-eks', {
+      version: eks.KubernetesVersion.V1_21,
+    });
+    const serviceAccount = cluster.addServiceAccount("eks-service-account");
+    const table = new dynamodb.Table(this, 'Table', {
+      partitionKey: { name: 'id', type: dynamodb.AttributeType.STRING }
```

Comment:
You can add a `tableName` property to force a more predictable table name: https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-dynamodb.Table.html